lib/ash_authentication/jwt/config.ex

```diff
@@ -48,6 +48,18 @@ defmodule AshAuthentication.Jwt.Config do
       &Joken.generate_jti/0,
       &validate_jti(&1, &2, &3, opts)
     )
+    |> maybe_add_tenant_claim(opts[:tenant])
+  end
+
+  defp maybe_add_tenant_claim(cfg, nil), do: cfg
+
+  defp maybe_add_tenant_claim(cfg, tenant) do
+    Config.add_claim(
```

Comment:
So the main thing missing here is that if there is a tenant claim, and no `opts[:tenant]` then it is invalid. How to we add this?

Reply:
Alright,  I remember the simpler answer. its "does the user resource support multi tenancy", and if so there is a tenant claim that must match, nil or otherwise.